migrationmanager/migrators/logs/migrations/000009_add_usage.down.sql

```diff
@@ -0,0 +1,2 @@
+DROP TABLE IF EXISTS signoz_logs.usage on cluster {{ .SIGNOZ_CLUSTER }}
```

Comment:
Did you test the down? This will probably fail if the individual queries must be terminated by `;` otherwise whole contents will be interpreted as a single query.

Reply:
Aah right, I tested the up but down I just ran the queries manually, have fixed it :)